ui/libproject/devstart.rb

```diff
@@ -40,6 +52,57 @@ def parse args
   end
 end
 
+# Command line parsing and run logic for deploy-ui command
+class DeployUI
```

Comment:
It looks like there's a fair amount of code copied from api/libproject/devstart.rb -- could you pull out a common superclass in workbench.rb? Or if that'd require too much work to separate out the creds_file stuff, please add cross-reference comments and next time we have to change one/both there's a reminder to consider refactoring and/or edit both places.